src/lib.rs

```diff
@@ -996,6 +1010,56 @@ impl Build {
             }
         }
 
+        let cudart = match &self.cudart {
+            Some(opt) => opt.as_str(),
+            None => "none",
+        };
+        if self.cuda && cudart != "none" {
```

Comment:
I think here you can probably just do `if let Some(cudart) = &self.cudart { ... }` because if someone specifies cudart then `self.cuda` is automatically set.

Reply:
If I `let Some(cudart) = &self.cudart`, I'll still have to check for "none". Because cudart being `None` and `Some("none")` are equivalent cases. But yes, I can omit `self.cuda`...

Reply:
Oh how come `Some("none")` is equivalent to `None`? (I thought that was accidental, didn't realize it was intentional)

Reply:
`self.cuda` is removed, added a commentary to remind that `Some("none")` is an option. I've also spotted error, I recognized "dynamic" instead of "shared."

Reply:
> I think here you can probably just do `if let Some(cudart) = &self.cudart { ... }` because if someone specifies cudart then `self.cuda` is automatically set.

Since `.cudart()` aims to mimic `--cudart`, it recognizes all `--cudart`'s options, {none|shared|static}. So that nvcc's users feel "home."